src/state/slices/common-selectors.ts

```diff
@@ -26,10 +26,10 @@ export const selectWalletSupportedChainIds = (state: ReduxState) =>
 export const selectWalletAccountIds = createDeepEqualOutputSelector(
   selectWalletId,
   (state: ReduxState) => state.portfolio.wallet.byId,
-  (state: ReduxState) => state.portfolio.hiddenAccountIds,
-  (walletId, walletById, hiddenAccountIds): AccountId[] => {
+  (state: ReduxState) => state.portfolio.enabledAccountIds,
+  (walletId, walletById, enabledAccountIds): AccountId[] => {
     const walletAccountIds = (walletId && walletById[walletId]) ?? []
-    return walletAccountIds.filter(accountId => !hiddenAccountIds.includes(accountId))
+    return walletAccountIds.filter(accountId => enabledAccountIds.includes(accountId))
```

Comment:
If we do add proper feature flag checks, this will cause a bunch of regressions across the app with the change from `!hiddenAccountIds.includes()` (non-breaking change as the array would be empty without hidden AccountIds) to `enabledAccountIds.includes()` (now assumes `enabledAccountIds`.

In the spirit of progressive, non-breaking enhancements here, we should probably check for feature flag enabled here as well and not run the predicate if the flag is off?

Reply:
Also see https://shapeshift-dao.sentry.io/issues/215187/?environment=localhost&query=is:unresolved&referrer=issue-stream&statsPeriod=1h&stream_index=0

Reply:
Yes, this is from the team toggling the feature on and off in production, not users. Apotheosis actaully created a PR to address this
https://github.com/shapeshift/web/pull/6840
But we closed it as it's only due to forward schema migration being forced backwards when toggling the feature off.

Ok, since this has come up twice I'll wrap everything in the feature flag in a follow up 👍

Reply:
Keep in mind the actions to enable accountIds will need to stay so toggling the feature on will show some accounts (not all hidden), so only checking the toggle here.

Reply:
On second thought, this will not make a difference once this PR goes into prod as the forward migration will have occurred for all users. I'll add a commit here to coalesce so a rollback will not rug all users in prod, and skip the feature flag here.